manifests/b/BinaryNinja/Demo/idk/BinaryNinja.Demo.installer.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: BinaryNinja.Demo
+PackageVersion: idk

```

```suggestion
PackageVersion: 1.0
```
When package version is null, always use some form of semantic version